src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -16,57 +16,57 @@ public static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Vector128Alignment;
 
-        public static void MatTimesSrc(bool tran, AlignedArray mat, AlignedArray src, AlignedArray dst, int crun) => SseUtils.MatTimesSrc(tran, mat, src, dst, crun);
+        public static void MatrixTimesSource(bool transpose, AlignedArray matrix, AlignedArray source, AlignedArray destination, int stride) => SseUtils.MatTimesSrc(transpose, matrix, source, destination, stride);
 
-        public static void MatTimesSrc(AlignedArray mat, int[] rgposSrc, AlignedArray srcValues,
-            int posMin, int iposMin, int iposLim, AlignedArray dst, int crun) => SseUtils.MatTimesSrc(mat, rgposSrc, srcValues, posMin, iposMin, iposLim, dst, crun);
+        public static void MatrixTimesSource(AlignedArray matrix, int[] rgposSrc, AlignedArray sourceValues,
+            int posMin, int iposMin, int iposLimit, AlignedArray destination, int stride) => SseUtils.MatTimesSrc(matrix, rgposSrc, sourceValues, posMin, iposMin, iposLimit, destination, stride);
 
-        public static void Add(float a, Span<float> dst) => SseUtils.Add(a, dst);
+        public static void Add(float value, Span<float> destination) => SseUtils.Add(value, destination);
```

Comment:
When something is being added it seems like sometimes we call it `value` and sometimes `addend`. And if we want to make them consistently `addend` maybe `value` should be more specific in other cases, such as `scale`. Eg.,
`Add(float addend, Span<float> destination)`
`ScaleAdd(float scale, float addend, Span<float> destination) `
`AddScale(float scale, ReadOnlySpan<float> addends, Span<float> destination, int count)`

But I defer to @tannergooding as I haven't given this previous thought and I don't know what the convention is. Just as long as we're consistent.

Reply:
I don't think that should stop merging this PR though.